Tests/NetworkLoggerPluginSpec.swift

```diff
@@ -74,37 +73,34 @@ final class NetworkLoggerPluginSpec: QuickSpec {
             expect(log).to(contain("Request: (invalid request)"))
         }
 
-        it("outputs the response data") {
-            let response = Response(statusCode: 200, data: "cool body".data(using: .utf8)!, response: HTTPURLResponse(url: URL(string: url(GitHub.zen))!, mimeType: nil, expectedContentLength: 0, textEncodingName: nil))
-            let result: Result<Moya.Response, MoyaError> = .success(response)
+        it("outputs the formatted request data") {
 
-            plugin.didReceive(result, target: GitHub.zen)
+            pluginWithRequestDataFormatter.willSend(TestBodyRequest(), target: GitHub.zen)
 
-            expect(log).to(contain("Response:"))
-            expect(log).to(contain("{ URL: https://api.github.com/zen }"))
-            expect(log).to(contain("cool body"))
+            expect(log).to(contain("Request: https://api.github.com/zen"))
+            expect(log).to(contain("Body: formatted request body"))
         }
 
-        it("outputs the formatted response data") {
+        it("outputs the response data") {
             let response = Response(statusCode: 200, data: "cool body".data(using: .utf8)!, response: HTTPURLResponse(url: URL(string: url(GitHub.zen))!, mimeType: nil, expectedContentLength: 0, textEncodingName: nil))
             let result: Result<Moya.Response, MoyaError> = .success(response)
 
-            pluginWithResponseDataFormatter.didReceive(result, target: GitHub.zen)
+            plugin.didReceive(result, target: GitHub.zen)
 
             expect(log).to(contain("Response:"))
             expect(log).to(contain("{ URL: https://api.github.com/zen }"))
-            expect(log).to(contain("formatted body"))
+            expect(log).to(contain("Response Body: cool body"))
         }
 
-        it("outputs the formatted request data") {
+        it("outputs the formatted response data") {
             let response = Response(statusCode: 200, data: "cool body".data(using: .utf8)!, response: HTTPURLResponse(url: URL(string: url(GitHub.zen))!, mimeType: nil, expectedContentLength: 0, textEncodingName: nil))
             let result: Result<Moya.Response, MoyaError> = .success(response)
 
-            pluginWithRequestDataFormatter.didReceive(result, target: GitHub.zen)
+            pluginWithResponseDataFormatter.didReceive(result, target: GitHub.zen)
 
             expect(log).to(contain("Response:"))
             expect(log).to(contain("{ URL: https://api.github.com/zen }"))
-            expect(log).to(contain("formatted request body"))
```

Comment:
There seemed to be a mismatch between the test's name and its content: it was supposed to check the requestDataFormatter, but acted like it was testing the response. So I fixed it by trully checking the request.